prowler/providers/common/audit_info.py

```diff
@@ -322,8 +322,9 @@ def set_azure_audit_info(self, arguments) -> Azure_Audit_Info:
             credential_scopes=region_config["credential_scopes"],
         )
 
-        if not arguments.get("only_logs"):
-            self.print_azure_credentials(azure_audit_info)
+        # this logic is being processed in general provider
+        # if not arguments.get("only_logs"):
+        #     self.print_azure_credentials(azure_audit_info)
```

Comment:
If this is not needed please remove it or include a `TODO` to remove it.

Reply:
Added